test-configs.yaml

```diff
@@ -1677,46 +1677,57 @@ test_configs:
   - device_type: imx6q-sabresd
     test_plans:
       - baseline
+      - baseline-nfs
 
   - device_type: imx6qp-sabresd
     test_plans:
       - baseline
+      - baseline-nfs
 
   - device_type: imx6sll-evk
     test_plans:
       - baseline
+      - baseline-nfs
```

Comment:
Similar timeout issue here: https://lavalab.nxp.com/scheduler/job/96431

Reply:
Hi gctucker,
There was one additional issue that got fixed yesterday.
The issue was w.r.t downloading big files (plain log files). Probably that was impacting the download of rfs?
The fix was to upgrade gunicorn to latest 20.0.4 for LAVA 2020.09. Remi helped in identifying and fixing the issue.

I have tried both the jobs and now I don't see any issue with the download.
However, there were issues with the boot up and I see issues with both the jobs. I have requested our team to look into the issue.
For imx6qp-sabresd, our team updated that the boot issue is a known issue and is fixed in the latest kernel.
Can you submit jobs for other boards as well please? If there are any issues, we will look into them and resolve the same.

Reply:
Jobs get submitted automatically every day on staging.kernelci.org so you should be able to check the results with the latest versions of mainline and linux-next.

Reply:
The kernel started but got stuck with an error: https://staging.kernelci.org/test/plan/id/5f902e26b432869aba729036/

Same as imx6qp-sabresd, please confirm if this can be fixed easily or if we need to wait for a kernel patch to land.